src/Symfony/Component/Form/FormRendererEngineInterface.php

```diff
@@ -127,8 +127,6 @@ public function getResourceHierarchyLevel(FormView $view, array $blockNameHierar
      * @param FormView $view      The view to render
      * @param mixed    $resource  The renderer resource
      * @param array    $variables The variables to pass to the template
-     *
-     * @return string
      */
-    public function renderBlock(FormView $view, mixed $resource, string $blockName, array $variables = []);
+    public function renderBlock(FormView $view, mixed $resource, string $blockName, array $variables = []): string;
```

Comment:
Add a native return type is a BC break. We cannot do that until 7.0 (and there is no chance we'll merge that in the 6.0.x patch release)

Reply:
alright @stof , I was not aware of that. Thanks for the docs link below!